internal/generator/vector/output/syslog/syslog.go

```diff
@@ -128,31 +121,19 @@ func Output(id string, o logging.OutputSpec, inputs []string, secret *corev1.Sec
 	}
 }
 
-func Encoding(id string, o logging.OutputSpec) Element {
+func Encoding(id string, o obs.OutputSpec) Element {
 	return SyslogEncoding{
-		ComponentID:  id,
-		RFC:          RFC(o.Syslog),
-		Facility:     Facility(o.Syslog),
-		Severity:     Severity(o.Syslog),
-		AppName:      AppName(o.Syslog),
-		ProcID:       ProcID(o.Syslog),
-		MsgID:        MsgID(o.Syslog),
-		Tag:          Tag(o.Syslog),
```

Comment:
What happened to the `Tag`, `LogSource`, and `PayloadKey`?

Reply:
> What happened to the `Tag`, `LogSource`, and `PayloadKey`?

I removed them based upon [a conversation](https://redhat-internal.slack.com/archives/C03744G7KE1/p1715180838236909?thread_ts=1714754608.783289&cid=C03744G7KE1) where I wish to move away from any special behavior, but I'm questioning if this was the right call.  I actually am wondering if it makes sense to not offer them but if we should be doing some transformation to fill in these fields based record content, or continue to offer them as-is and allow the admin to choose. We should discuss.